test/test_class1_allele_specific_cv_and_train_command.py

```diff
@@ -62,6 +68,8 @@ def write_json(payload, filename):
         "--verbose",
         "--num-local-threads", "1",
     ]
+    if KUBEFACE_INSTALLED:
```

Comment:
Can you add a comment here explaining why having kubeface implies a local storage prefix?